docs/tutorials/getting-started-with-csharp/WeatherMicroservice/README.md

```diff
@@ -0,0 +1,27 @@
+C# Microservices Sample
+================
+
+This sample is created during the [Creating Microservices hosted in Docker](../microservices.html)
```

Comment:
Two thoughts.

- I'd first introduce the sample for what it is, not the fact that it is associated with some content. The "key features" content seems like the intro to me. I'd then follow up with the tutorial link.
- "during" seems like an odd term to describe its association. I was expecting something for like: "You can read the [Underwater basket weaving made easy](https://underwaterbasketweaving.com) tutorial for step-by-step instructions on creating this sample."